### DISCUSSION TITLE
ubuntu 24.04 installation error with pip (used pipx instead)

### DISCUSSION THREAD

**Discussion by stannous** [mbridak/FieldDayLogger#44] [General]
Hello,

Just found your fine program.  Thanks!
One little problem I had installing on an ubuntu 24.04 laptop was an error installing with pip.

$ pip install fdlogger
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
  <snipped>
  
I used pipx instead and it everything ran just fine.

$ sudo apt install pipx
Reading package lists... Done
<snipped>

$ pipx install fdlogger
  installed package fdlogger 24.5.25, installed using Python 3.12.3
  These apps are now globally available
    - fdlogger
done! ✨ 🌟 ✨

Thanks again,
Sam Tannous, KJ4VWG

**Comment by mbridak** [comment]
Thanks. I'll make a note in the readme when I get back home.

**Reply by gibbsjj** [reply]
I just installed on Ubuntu 24.04.1 and also had to use pipx. After that I was getting "could not find the Qt platform plugin 'wayland'". 

The fix for me was: 

export QT_QPA_PLATFORM=xcb

After that, it starts and runs as expected. Thanks for sharing your software!

73
